README.md

```diff
@@ -1480,6 +1480,10 @@ For each **table** in `tables` property:
 14. `deleteFields` / `addFields` - delete (or add with a default value), a field at the specified path for all JSON entries in the table
 15. `populateJsonWithId` - This schema.json entry and the disable option is no longer supported. The primary key is always copied into `jsonb->'id'` on each insert and update.
 16. `pkColumnName` - No longer supported. The name of the primary key column is always `id` and is copied into `jsonb->'id'` in each insert and update. The method PostgresClient.setIdField(String) no longer exists.
+17. `withOptimisticLocking` - Creates insert/update trigger to auto populate/update `_version` field in the json
+    * `off` Optimistic Locking is disabled. The trigger does nothing
+    * `logOnConflict` Optimistic Locking is enabled. Version conflict info is logged as a warning message
+    * `failOnConflict` Optimistic Locking is enabled. Version conflict will fail the transaction
```

Comment:
We have this paragraph and the paragraph at line 1818 below about optimistic locking. Both are lacking some information. To avoid repetition I suggest to only list the options here

    17. `withOptimisticLocking` - `off` (default), `logOnConflict`, or `failOnConflict`, for details see [optimistic locking section](#optimistic-locking) below

and explain all details below.

Reply:
> We have this paragraph and the paragraph at line 1818 below about optimistic locking. Both are lacking some information. To avoid repetition I suggest to only list the options here
> 
> ```
> 17. `withOptimisticLocking` - `off` (default), `logOnConflict`, or `failOnConflict`, for details see [optimistic locking section](#optimistic-locking) below
> ```
> 
> and explain all details below.

Done.